tiled/client/catalog.py

```diff
@@ -21,6 +23,26 @@
 from .utils import handle_error
 
 
+def get_token(uri):
+    username = input("Username: ")
+    password = getpass.getpass()
+    form_data = {'grant_type': 'password',
+                 'username': username,
+                 'password': password}
+    response = httpx.post(uri + '/token', data=form_data)
+    return 'Bearer ' + json.loads(response.text)['access_token']
```

Comment:
Minor thing: we might want to the token alone. Can Bearer be tacked on later, when it is placed in the header?

Reply:
Oh, and `response.json()` would be more idiomatic here.